### PR TITLE
Work towards a better introduction flow

### PR DESCRIPTION
**Pull request by henriquegemignani**
- Specific tab for Multiworld
- Introduction tab for each game
- Game generation is next to the introduction tab of each game

This new UI looks terrible because I haven't polished at all, but I want feedback on how it works.

### CODE REVIEW

File: randovania/gui/lib/preset_tree_widget.py [hunk 1]
```diff
@@ -126,24 +112,15 @@ def select_preset(self, preset: VersionedPreset):
...
-        else:
+        if uid is not None:
             with self.options as options:
                 options.set_preset_uuid_hidden(uid, not new_state)
 
     def on_item_expanded(self, item):
         self._on_item_new_state(item, True)
 
     def on_item_collapsed(self, item):
+        self.root_tree_item.setExpanded(True)
```
Comment (duncathan) [inline randovania/gui/lib/preset_tree_widget.py:125]: a comment here explaining the hack might be nice

File: randovania/gui/main_window.py [hunk 2]
```diff
@@ -245,7 +265,7 @@ async def _initialize_post_show_body(self):
                                                       self._options)
 
         logging.info("Running GenerateSeedTab.setup_ui")
-        self.tab_create_seed.setup_ui(self, self, self._options)
+        # self.tab_create_seed.setup_ui(self, self, self._options)
```
Comment (duncathan) [inline randovania/gui/main_window.py:268]: if this is being commented, the log should be too

**Review by duncathan — CHANGES_REQUESTED** [review]
on the functional side, I think the Introduction tab should include a button to generate a seed using the default preset for each game. that way absolute newcomers can just get right into it. obviously the aesthetics need some substantial improvements too lol

**Comment by henriquegemignani** [comment]
> on the functional side, I think the Introduction tab should include a button to generate a seed using the default preset for each game. that way absolute newcomers can just get right into it. obviously the aesthetics need some substantial improvements too lol

I'm unsure how much of the introduction tab should be freestyle per game and how much we expect some things to allow for code sharing.

**Review by duncathan — COMMENTED** [review]
nothing seems immediately wrong

**Review by duncathan — APPROVED** [review]
lots of improvements yet to make, but I'm gonna go ahead and merge so we can start collecting feedback ASAP. some immediate things:
 - write text for TODOs
 - declutter the main window tabs. help, about, and changelog can probably all be moved to a dropdown menu
 - consider changing the game select in the Games tab from even more tabs to another method. three layers of tabs is a bit much imo
 - investigate aligning the flow layout somehow
 - rework the welcome tab
 - have the game cover images change when hovered, to indicate they're clickable

**Pull request merged**